src/model/transaction/__tests__/insertFragmentIntoContentState-test.js

```diff
@@ -107,3 +118,277 @@ test('must apply multiblock fragments', () => {
     ]),
   );
 });
+
+test('must apply a single ContentBlockNode from fragment', () => {
+  const initialSelection = SelectionState.createEmpty('A');
+  const initialContent = contentState.set(
+    'blockMap',
+    createContentBlockNodeFragment([
+      {
+        key: 'A',
+        text: '',
+        data: null,
+      },
+    ]),
+  );
+
+  assertInsertFragmentIntoContentState(
+    createContentBlockNodeFragment([
+      {
+        key: 'B',
+        text: 'some text',
+      },
+    ]),
+    initialSelection,
+    initialContent,
+  );
+});
+
+test('must apply ContentBlockNode fragments', () => {
```

Comment:
Lets rename this to something more meaningful like,

`must be able to insert fragment of ContentBlockNodes`